### DISCUSSION TITLE
It seems that the graph in the left panel is a bit inaccurate?

### DISCUSSION THREAD

**Discussion by iCarl** [jesec/flood#293] [General]
I'm running Flood in systemd on qbittorrent, and while the speed value in "Download Speed" seems regular, the graph and speed in the left panel seems quite irregular, what could be the cause of this?

https://user-images.githubusercontent.com/4193514/116152377-21853900-a6e6-11eb-98c7-313566c1dd87.mov

**Comment by jesec** [comment]
Your qBittorrent speed fluctuates.

**Comment by iCarl** [comment]
It could be that, but it doesn't look like it does.
The torrent at the top is the only one that is downloading.

https://user-images.githubusercontent.com/4193514/116206440-17435900-a73f-11eb-8c16-2af142f933fc.mov

**Reply by jesec** [reply]
Torrents have their own speeds, but there are also global speeds. 

They come from separate places, and Flood fetches them separately. 

If you use qBittorrent GUI, you will notice such deviance as well.
